### PR TITLE
Added a blog-post on release of cake-rider

### PR DESCRIPTION
**Pull request by nils-a**
@gep13 @pascalberger please let me know what you think. Mainly I've only "copied" the information from the readme of cake-rider.

**Things that need to be changed, after the release and before the merge:**
 - [x] I "named" post 2021-01-01, so we probably need to change that when we know the date
 - [x] There's a screenshot "from the marketplace" near the end, wich is faked for now. When we've done the first release that image must be replaced.
 - [x] There's a link (placeholder) to the plugin in the marketplace, that needs to be set correctly.
 - [x] Fixup the multiple commits I've made

### CODE REVIEW

File: input/blog/2021-01-01-cake-for-rider.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+![Run configurations](/assets/img/cake-rider/runConfigurations.png)
+
+The configurations can either be created from an existing Cake task, using the tool window or 
+created manually using the run configuration editor:
+
+![Run configuration editor](/assets/img/cake-rider/runConfiguration-editor.png)
+
+### Getting started
+
+In Rider, got to File -> Settings -> Plugins -> Marketplace and search for *Cake Rider*
```
Comment (gep13) [inline input/blog/2021-01-01-cake-for-rider.md:37]:
```suggestion
In Rider, go to File -> Settings -> Plugins -> Marketplace and search for *Cake Rider*
```

**Review by gep13 — CHANGES_REQUESTED** [review]
Just a couple small suggestions, but on the whole, this looks good to me!

**Comment by nils-a** [comment]
@gep13 I implemented the changes.
Additionally I added a link to the plugin in the marketplace (obviously it's not the final link...)

**Comment by nils-a** [comment]
@gep13 you wrote in https://github.com/cake-build/cake-rider/issues/5 :

> Mention Anna's original extension, and how she is now working with us to create this official extension.

I totally forgot to add that. Will add that, too.

**Comment by gep13** [comment]
Sounds like a plan!

**Comment by nils-a** [comment]
@gep13 @pascalberger I have replaced the image from the marketplace and set the url correctly. Could you have a (final) look at this?

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by nils-a** [comment]
I did one last change: I changed the date of the post to today, as today Cake for Rider 0.1.0 has been released and approved. 

So, now I'm fine with merging this in. :shipit:

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@nils-a thanks for getting this written, and obviously for all the work that you have done on getting the extension created in the first place!

**Pull request merged**